openmls/src/prelude.rs

```diff
@@ -2,17 +2,10 @@
 //! Include this to get access to all the public functions of OpenMLS.
 
 // MlsGroup
-pub use crate::group::{
-    proposals::{ProposalStore, QueuedProposal},
-    EmptyInputError, GroupEpoch, GroupId, IncomingWireFormatPolicy, InnerState,
-    InvalidMessageError, MlsGroup, MlsGroupConfig, MlsGroupError, OutgoingWireFormatPolicy,
-    WireFormatPolicy, ALL_VALID_WIRE_FORMAT_POLICIES,
-};
+pub use crate::group::*;
```

Comment:
Shouldn't we export these things explicitly rather than via wildcard?

Reply:
We export the right ones in the module now since this has been cleaned up. There is no need to be more fine-grained here anymore.